lib/src/arg_parser.dart

```diff
@@ -29,15 +30,26 @@ class ArgParser {
   /// arguments.
   final bool allowTrailingOptions;
 
+  /// Whether or not this parser treats unrecognized options as non-option
+  /// arguments.
+  bool get allowsAnything => false;
+
   /// Creates a new ArgParser.
   ///
   /// If [allowTrailingOptions] is set, the parser will continue parsing even
   /// after it finds an argument that is neither an option nor a command.
   /// This allows options to be specified after regular arguments. Defaults to
   /// `false`.
   factory ArgParser({bool allowTrailingOptions: false}) =>
-      new ArgParser._(<String, Option>{}, <String, ArgParser>{},
-          allowTrailingOptions: allowTrailingOptions);
+      new ArgParser._({}, {}, allowTrailingOptions: allowTrailingOptions);
+
+  /// Creates a new ArgParser that treats *all input* as non-option arguments.
+  ///
+  /// This is intended to allow arguments to be passed through to child
+  /// processes without needing to be redefined in the parent.
+  ///
+  /// Options may not be defined for this parser.
+  factory ArgParser.allowAnything() = AllowAnythingParser;
```

Comment:
Instead of exposing a constructor for a subclass that basically says all of its methods can't be used, how about adding `addUnparsedCommand(String name)` to ArgParser. That would implicitly create an AllowAnythingParser under the hood but not give access to it.